tensorflow-examples/src/main/java/org/tensorflow/model/examples/cnn/fastrcnn/FasterRcnnInception.java

```diff
@@ -0,0 +1,342 @@
+            "airplane",
+            "bus",
+            "train",
+            "truck",
+            "boat",
+            "traffic light",
+            "fire hydrant",
+            "street sign",
+            "stop sign",
+            "parking meter",
+            "bench",
+            "bird",
+            "cat",
+            "dog",
+            "horse",
+            "sheep",
+            "cow",
+            "elephant",
+            "bear",
+            "zebra",
+            "giraffe",
+            "hat",
+            "backpack",
+            "umbrella",
+            "shoe",
+            "eye glasses",
+            "handbag",
+            "tie",
+            "suitcase",
+            "frisbee",
+            "skis",
+            "snowboard",
+            "sports ball",
+            "kite",
+            "baseball bat",
+            "baseball glove",
+            "skateboard",
+            "surfboard",
+            "tennis racket",
+            "bottle",
+            "plate",
+            "wine glass",
+            "cup",
+            "fork",
+            "knife",
+            "spoon",
+            "bowl",
+            "banana",
+            "apple",
+            "sandwich",
+            "orange",
+            "broccoli",
+            "carrot",
+            "hot dog",
+            "pizza",
+            "donut",
+            "cake",
+            "chair",
+            "couch",
+            "potted plant",
+            "bed",
+            "mirror",
+            "dining table",
+            "window",
+            "desk",
+            "toilet",
+            "door",
+            "tv",
+            "laptop",
+            "mouse",
+            "remote",
+            "keyboard",
+            "cell phone",
+            "microwave",
+            "oven",
+            "toaster",
+            "sink",
+            "refrigerator",
+            "blender",
+            "book",
+            "clock",
+            "vase",
+            "scissors",
+            "teddy bear",
+            "hair drier",
+            "toothbrush",
+            "hair brush"
+    };
+
+    public static void main(String[] params) {
+
+        if (params.length != 2 ){
+            throw new IllegalArgumentException("Exactly 2 parameters required !");
+        }
+
+        //my test image
+        String outputImagePath  = params[1];
+        String imagePath = params[0];
+
+        // get path to model folder
+        String modelPath = "models/faster_rcnn_inception_resnet_v2_1024x1024";
+        // load saved model
+        SavedModelBundle model = SavedModelBundle.load(modelPath, "serve");
+
+        //create a map of the COCO 2017 labels
+        TreeMap<Float, String> cocoTreeMap = new TreeMap<>();
+        float cocoCount = 0;
+        for (String cocoLabel : cocoLabels) {
+            cocoTreeMap.put(cocoCount, cocoLabel);
+            cocoCount++;
+        }
+        try (Graph g = new Graph(); Session s = new Session(g)) {
+
+            Ops tf = Ops.create(g);
+
+
+
+
```

Comment:
Nit: that's a lot of empty lines, can you please reformat?

Reply:
Just tidied up abit